lib/osf-components/addon/components/sort-arrow/template.hbs

```diff
@@ -0,0 +1,17 @@
+<span local-class='sort-arrow-container'>
+    <div
+        role='button'
```

Comment:
I'd like to see this turned into a standard `<button>` or a `<Button>`, but let's save this for a later JIRA ticket